lib/ex_doc/formatter/html.ex

```diff
@@ -128,7 +128,7 @@ defmodule ExDoc.Formatter.HTML do
   def ast_to_html(binary) when is_binary(binary), do: Templates.h(binary)
 
   def ast_to_html({tag, attrs, ast, %{verbatim: true}}) do
-    [text] = ast
+    text = Enum.join(ast, "")
     ["<#{tag}", ast_attributes_to_html(attrs), ">", text, "</#{tag}>"]
```

Comment:
this function returns an iolist so there's no need to join the elements:

```suggestion
    ["<#{tag}", ast_attributes_to_html(attrs), ">", ast, "</#{tag}>"]
```

Reply:
You are right